modules/openthread/platform/radio.c

```diff
@@ -385,6 +393,20 @@ void transmit_message(struct k_work *tx_job)
 	radio_api->set_channel(radio_dev, channel);
 	radio_api->set_txpower(radio_dev, get_transmit_power_for_channel(channel));
 
+#if defined(CONFIG_OPENTHREAD_TIME_SYNC)
+	if (sTransmitFrame.mInfo.mTxInfo.mIeInfo->mTimeIeOffset != 0) {
+		*(sTransmitFrame.mPsdu + sTransmitFrame.mInfo.mTxInfo.mIeInfo->mTimeIeOffset) =
```

Comment:
@DaveLacerteHQ In my [previous comment](https://github.com/zephyrproject-rtos/zephyr/pull/70758#discussion_r1540664551) I was referring to this line, not the line below where the pointer is initialized.

Reply:
You are right. Sorry for the confusion. Corrected.